tests/HelloWorldWPF/MainWindow.xaml

```diff
@@ -74,6 +74,15 @@
             </Button.Template>
         </Button>
         <TextBlock Text="{lex:Loc {Binding tenum, StringFormat=TestEnum_{0}}}"></TextBlock>
+        <TextBlock>
+            <TextBlock.Text>
+                <lex:Loc>
+                    <MultiBinding StringFormat="{}{0}">
+                        <Binding Path="tenum" StringFormat="TestEnum_{0}"/>
+                    </MultiBinding>
```

Comment:
It is not work as expected. I think it shold be like that:
``` xaml
<MultiBinding StringFormat="{}TestEnum_{0}">
    <Binding Path="tenum" />
</MultiBinding>
```
[Explanation](https://stackoverflow.com/a/15530867/6862052)
And yes, my version looks weird :)

Reply:
> It is not work as expected. I think it shold be like that:
> 
> ```
> <MultiBinding StringFormat="{}TestEnum_{0}">
>     <Binding Path="tenum" />
> </MultiBinding>
> ```
> 
> [Explanation](https://stackoverflow.com/a/15530867/6862052)
> And yes, my version looks weird :)

@karnah
I change the example to avoid this MultiBinding StringFormat Bug. Thanks for the Explanation.